### PR TITLE
Camel case columns on data import

### PR DESCRIPTION
**Pull request by tiagojsag**
https://vizzuality.atlassian.net/browse/LANDGRIF-354

### CODE REVIEW

File: api/test/h3-data/mocks/create-fake-h3-data.ts [hunk 1]
```diff
@@ -13,13 +14,16 @@ export const createFakeH3Data = async (
   indicatorId?: string,
   year: number = 2000,
 ): Promise<H3Data> => {
+  const formattedTableName: string = snakeCase(h3TableName);
+  const formattedColumnName: string = camelCase(h3ColumnName);
+
```
Comment (alexeh) [inline api/test/h3-data/mocks/create-fake-h3-data.ts:19]: nice

File: api/test/h3-data/mocks/create-fake-h3-data.ts [hunk 2]
```diff
@@ -41,8 +45,8 @@ export const createFakeH3Data = async (
 };
 
 export const dropFakeH3Data = async (h3TableNames: string[]): Promise<void> => {
-  for (const h3tableName of h3TableNames) {
-    await getManager().query(`DROP TABLE IF EXISTS ${h3tableName};`);
+  for (const h3TableName of h3TableNames) {
+    await getManager().query(`DROP TABLE IF EXISTS ${snakeCase(h3TableName)};`);
```
Comment (alexeh) [inline api/test/h3-data/mocks/create-fake-h3-data.ts:49]: Shouldn't this be:

```suggestion
    await getManager().query(`DROP TABLE IF EXISTS ${formattedTableName)};`);
```

**Review by alexeh — APPROVED** [review]
LGTM, thank you! Just a minor comment

File: api/test/h3-data/mocks/create-fake-h3-data.ts (hunk 2 above)
Comment (tiagojsag) [reply api/test/h3-data/mocks/create-fake-h3-data.ts:49]: formattedTableName does not exist in the scope of this function

**Pull request merged**